liquibase-extension-testing/src/main/groovy/liquibase/extension/testing/command/CommandTests.groovy

```diff
@@ -460,7 +460,6 @@ Long Description: ${commandDefinition.getLongDescription() ?: "NOT SET"}
                         throw new ComparisonFailure("$outputDescription does not contain expected", expectedOutputCheck, fullOutput)
                     }
                 } else if (expectedOutputCheck instanceof Pattern) {
-                    String patternString = StringUtil.standardizeLineEndings(StringUtil.trimToEmpty(((Pattern) expectedOutputCheck).pattern()))
```

Comment:
Why was this line removed, please?

Reply:
The result was never used

Reply:
That's as good a reason as any to remove that line! Resolving this conversation.